packages/create-astro/src/index.ts

```diff
@@ -174,7 +174,7 @@ export async function main() {
         return indent + importStatements.join('\n');
       })
       .replace(/^(\s*)<!-- ASTRO:COMPONENT_MARKUP -->/gm, (_, indent) => {
-        return components.map((ln) => indent + ln).join('\n');
```

Comment:
This one should be safe to keep, these aren't authored components. This just modifies our starter template.

Reply:
Yes. Thank you for explaining. These can be kept.